addons/event/static/src/scss/event_full_page_ticket_responsive_html_report.scss

```diff
@@ -14,6 +14,11 @@
             }
         }
     }
+
+    .o_event_download_ticket_btn:hover {
+        border-color: #374151;
```

Comment:
It's usually a bad practice to hardcode colors, as they could conflict with the theme used on this database.

Surely we can find something that achieves what we want by using "btn-primary, btn-secondary, btn-light, ..." ?

Reply:
Hello @awa-odoo :wave: 

I thought that too that hardcoding the scss color is bad but as we can see this is the preview page 
it will always be white no matter what the theme is in the database
Ref image - https://i.imgur.com/aVZF9zV.png
In the preview page, the `btn-primary` and some other classes are not working
`btn-link` is working on the preview page but it doesn't look good as this is one so that's the main reason of writing this hardcoded scss :sweat_smile: 

Thanks!!

Reply:
Oh yes, because those templates are supposed to be for PDF, so they don't have the full bootstrap and theme capabilities, that makes sense.

Alright, we can keep it then, thanks for the explanation 🙂